templates/terminating-gateways-deployment.yaml

```diff
@@ -0,0 +1,324 @@
+{{- if .Values.terminatingGateways.enabled }}
```

Comment:
Can we use `len(.Values.terminatingGateways.gateways)` instead?

Reply:
I was going to have one gateway defined automatically in the `gateways` field as an example. Then someone could enable gateways and get a gateway without doing any extra work. It should also simplify the explanation of required gateway fields-- though I suppose we'll still have to explain the `extraVolumes` field so maybe that's a wash.